docs/topics/con_missions-and-cloud-native-development-on-openshift.adoc

```diff
@@ -1,35 +1,29 @@
 [id='missions-and-cloud-native-development-on-openshift_{context}']
-= Missions and cloud-native development on OpenShift
+= Example applications and cloud-native development on OpenShift
 
-When developing applications on OpenShift, you can use missions and boosters to kickstart your development.
+When developing applications on OpenShift, you can use example applications to kickstart your development.
 
 [discrete]
-== Missions
+== Example applications
 
-Missions are working applications that showcase different fundamental pieces of building cloud native applications and services.
+Examples are working applications that showcase different fundamental pieces of building cloud native applications and services.
 
-A mission implements a link:http://microservices.io/patterns/microservices.html[Microservice pattern] such as:
+The examples implement a link:http://microservices.io/patterns/microservices.html[Microservice pattern] such as:
 
 * Creating REST APIs
 * Interoperating with a database
 * Implementing the Health Check pattern
 
-You can use missions for a variety of purposes:
+You can use the examples for a variety of purposes:
 
 * A proof of technology demonstration
 * A teaching tool, or a sandbox for understanding how to develop applications for your project
 * They can also be updated or extended for your own use case
```

Comment:
This list is not parallel. Consider rephrasing as:
The examples that can be updated or extended according to your requirements can:
* Demonstrate a functionality
* Act as a teaching tool or a sandbox for understanding how to develop applications for your project